src/main/resources/application.yaml

```diff
@@ -186,13 +186,15 @@ config:
 payerFundTransfer:
   tenant:
     payer: "wakanda"
+    payee: "jupiter"
```

Comment:
use payee 1 instead of payee. It can create confusion when the same config is used with  diff name